pom.xml

```diff
@@ -246,6 +246,12 @@
             </dependency>
 
             <!-- test dependencies -->
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.13.2</version>
```

Comment:
It seems we are using version 5. Why do we need to declare this version 4?

Reply:
I don't think the `junit-bom` will bring an up-to-date version of that dependency since that is JUnit 4.

Reply:
The junit-bom brings that through the junit-vintage-engine, and we still have tests that use JUnit 4.